src/display/annotation_layer.js

```diff
@@ -197,6 +201,52 @@ class AnnotationElement {
     return AnnotationElement._hasPopupData(this.data);
   }
 
+  update(params) {
```

Comment:
Given where/how this is being used, how about making that clearer in the name (to help future readers of the code)?
```suggestion
  updateEdited(params) {
```